ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/host/MessageInterceptorInstaller.java

```diff
@@ -37,7 +37,7 @@ public <T> void install(final MessageInterceptorDescriptor<T> interceptorDescrip
    * Registers the passed interceptor in the SCION Microfrontend Platform. Intercepted messages are delegated to the passed callback.
    */
   private <T> void registerInterceptor(final JavaCallback interceptorCallback, final MessageInterceptorDescriptor<T> interceptorDescriptor,

```

Comment:
Let's format the parameter declarations to go one below another, aligned.

Same below.